.cirrus.yml

```diff
@@ -182,9 +182,47 @@ ruling_win_task:
     - cd its/ruling
     - mvn package -Pit-ruling -Dsonar.runtimeVersion=LATEST_RELEASE[9.9] -Dmaven.test.redirectTestOutputToFile=false -B -e -V -Dparallel=methods -DuseUnlimitedThreads=true
   cleanup_before_cache_script: cleanup_maven_repository
+
+autoscan_task:
+  depends_on:
+    - build
+  <<: *ONLY_SONARSOURCE_QA
+  eks_container:
+    <<: *CONTAINER_DEFINITION
+    image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-latest
+    cpu: 14
+    memory: 6G
+  maven_cache:
+    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
+  submodules_script:
+    - git submodule update --init --recursive
+  ruling_script:
+    - source cirrus-env QA
+    - source set_maven_build_version $BUILD_NUMBER
+    - mvn clean compile --projects java-checks-test-sources
+    - cd its/autoscan
+    - mvn package -Pit-autoscan -Dsonar.runtimeVersion=LATEST_RELEASE[9.9] -Dmaven.test.redirectTestOutputToFile=false -B -e -V -Dparallel=methods -DuseUnlimitedThreads=true
+  cleanup_before_cache_script: cleanup_maven_repository
   on_failure:
     actual_artifacts:
-      path: "${CIRRUS_WORKING_DIR}/its/ruling/target/actual/**/*"
+      path: "${CIRRUS_WORKING_DIR}/its/autoscan/target/actual/**/*"
+
+autoscan_win_task:
```

Comment:
Do we need the autoscan test on Windows? We have the ruling running on Windows already and e.g. plugin is also not being run on Windows, so I wonder if autoscan would be enough on Linux.

Reply:
Indeed, I am not sure if expect a different behavior for the 2 platforms. I will remove it for now